### ISSUE TITLE
"TypeError: Cannot read property 'flags' of null" with ng-template inside i18n directive

### ISSUE THREAD

**Issue by henry-alakazhang** [angular/angular#35746]
# 🐞 bug report

### Affected Package

 The issue is caused by package @angular/core or @angular/localize (I think)

### Is this a regression?

 Yes, this doesn't occur with enableIvy false

### Description
If you place an `ng-template` inside an `i18n` tag and put any other element after, a runtime error is thrown and the template + any related ngIfElse are not rendered

## 🔬 Minimal Reproduction
```html
<div i18n>
  <!-- this hr is fine -->
  <hr />
  <ng-container *ngIf="true; else hello">
    Welcome!
  </ng-container>
  <ng-template #hello>
    Hello there!
  </ng-template>
  <!-- removing this hr fixes the issue -->
  <hr />
</div>
```

note that putting text after the `</ng-template>` is fine, it's only other elements

## 🔥 Exception or Error
```
main.ts:12 TypeError: Cannot read property 'flags' of null
    at removeNode (core.js:35406)
    at i18nEndFirstPass (core.js:35096)
    at Module.ɵɵi18nEnd (core.js:35068)
    at AppComponent_Template (app.component.ts:14)
    at executeTemplate (core.js:11930)
    at renderView (core.js:11716)
    at renderComponent (core.js:13232)
    at renderChildComponents (core.js:11519)
    at renderView (core.js:11742)
    at ComponentFactory$1.create (core.js:33897)
```

or (firefox)

```
ERROR TypeError: "removedPhTNode is null"
    Angular 3
        removeNode
        i18nEndFirstPass
        i18nEnd
    AppComponent_Template app.component.ts:14
    Angular 20
```

The two `hr` elements are rendered but the container / template are not:

![image](https://user-images.githubusercontent.com/8922125/75517646-4edb4600-5a53-11ea-833d-fb198018cf25.png)

## 🌍  Your Environment

**Angular Version:**
<pre><code>
$ ng version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 9.0.4
Node: 12.4.0
OS: linux x64

Angular: 9.0.4
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, localize, platform-browser
... platform-browser-dynamic, router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.900.4
@angular-devkit/build-angular     0.900.4
@angular-devkit/build-optimizer   0.900.4
@angular-devkit/build-webpack     0.900.4
@angular-devkit/core              9.0.4
@angular-devkit/schematics        9.0.4
@ngtools/webpack                  9.0.4
@schematics/angular               9.0.4
@schematics/update                0.900.4
rxjs                              6.5.4
typescript                        3.7.5
webpack                           4.41.2
    
Done in 0.44s.
</code></pre>

**Anything else relevant?**
`ng xi18n` is able to extract the text fine.

This can be definitely be worked around by rearranging the text and i18n tag (and probably should be, since it can lead to more confusing "translateable" text), but it still seems like a bug.

**Comment by AndrewKushnir** [comment]
Hi @henry-alakazhang, thanks for reporting the issue and providing a repro. We've found the origin of the problem and I created a PR #35758 to resolve it. Thank you.

**Comment by aboyton** [comment]
Thanks for the quick fix. What’s the release timeline look for a change like this? Do you cut a patch release weekly?

**Comment by AndrewKushnir** [comment]
@aboyton, the mentioned PR was merged and the fix will be included into the `9.0.5` version (that should be available next week approximately). I will close this ticket for now. Feel free to reopen if the problem still exists in `9.0.5` version. Thank you.
